jumanji/environments/routing/macvrp/reward.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+
+import chex
+import jax
+import numpy as np
+
+from jumanji.environments.routing.macvrp.types import State
+
+
+class RewardFn(abc.ABC):
+    def __init__(self, num_vechicles: int, num_customers: int, map_max: int) -> None:
+        self.num_vehicles = num_vechicles
+        self.num_customers = num_customers
+        self.map_max = map_max
+        super().__init__()
```

Comment:
No super call is needed for abstract classes. 
```suggestion
```

Reply:
Updated.